micro-ui/web/packages/modules/workbench/src/components/icons/DustbinIcon.js

```
@@ -0,0 +1,10 @@
+import React from "react";
+import { PRIMARY_COLOR } from "../../utils";
+export const DustbinIcon = ({width, height}) => (
+  <svg width={width || "12"} height={height || "16"} viewBox="0 0 12 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)